app/views/dashboards/_table_content.html.erb

```diff
@@ -41,10 +41,8 @@
               <td><%= link_to "Add", add_patient_path(current_user, patient), method: :patch, remote: true %></td>
             <% end %>
           <% end %>
+          <td><%= link_to call_glyphicon, new_patient_call_path(patient), class:"call_button", remote: true %> </td>
```

Comment:
Same thing here, would you mind slapping a class `call-{phone-number}` on this link? e.g. `call-123-123-1234`